_data/media.yml

```
@@ -0,0 +1,8 @@
+- extension: mp3
+  mimeType: mpeg
+- extension: mov
+  mimeType: quicktime
+- extension: avi
+  mimeType: x-msvideo
+- extension: mkv
+  mimeType: x-matroska
```

Nit: Consider adding more [common types](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types) right away

```suggestion
  mimeType: x-matroska
- extension: ogv
  mimeType: ogg
- extension: weba
  mimeType: webm
- extension: 3gp
  mimeType: 3gpp
- extension: 3g2
  mimeType: 3gpp2
- extension: mid
  mimeType: midi
```